app/models/petition.rb

```diff
@@ -100,6 +102,10 @@ class Petition < ActiveRecord::Base
   scope :with_response, -> { where.not(response_summary: nil, response: nil) }
   scope :with_debate_outcome, -> { joins(:debate_outcome) }
 
+  scope :without_debate_date, -> { where.not(id: DebateOutcome.select(:petition_id).uniq) }
```

Comment:
Oh! I had no idea it would use a subquery.  I thought it would do two queries.  That's really neat, thanks for pointing it out @otlaitil!  Ignore me then, we can leave the implementation as-is.  

Now I have to go work out if this is PG specific or something AR does now whenever it can.